lib/ParamedicLog.js

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env node
+
+/* jshint node: true */
+
+"use strict";
+
+var shelljs  = require("shelljs");
+var fs       = require("fs");
+var path     = require("path");
+var util     = require('./utils').utilities;
+var logger   = require('./utils').logger;
+
+
+function ParamedicLog(platform, appPath, logPath){
+    this.platform = platform;
+    this.appPath = appPath;
+    this.logPath = logPath;
+}
+
+ParamedicLog.prototype.logIOS = function (appPath) {
+    // We need to print out the system log for the simulator app. In order to figure
+    // out the path to that file, we need to find the ID of the simulator running
+    // mobilespec
+
+    // First, figure out the simulator that ran mobilespec. "cordova run"" just chooses
```

Comment:
I find this to be very fragile code because it depends on internal implementation details of ```cordova run --list```. Those could well change in the future. I think it  might be better to just copy the logic that deals with listing the simulators in this project.
What do you think ?